### PR TITLE
Changed the title to let people know they can contribute

### LINKED ISSUE #1: Create website linking to simple donuts in each framework

### PR DESCRIPTION
**Pull request by AniaKB**
I'm still working on it. 

Closes #1

### CODE REVIEW

**Comment by etdr** [comment]
Looks good to me

**Comment by AniaKB** [comment]
I've added the HTML and Stylesheet.

File: index.html [hunk 1]
```diff
@@ -1,10 +1,50 @@
...
 <html>
   <head>
+    <link rel="stylesheet" href="style.css">
     <title>Simple Donut</title>
   </head>
   <body>
-    <h1>Coming Soon</h1>
+    <header>
+      <h1>What is Simple Donut?</h1>
+      <p>Simple Donut is a percentage chart that can be toggled from pie to donut.
```
Comment (TheJaredWilcurt) [inline index.html]: This way it matches the next two lines
```suggestion
      <p>
        Simple Donut is a percentage chart that can be toggled from pie to donut.
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+td {
+  border-style: dotted;
+  border-width: 2px;
+}
+
+p {
+  width: 550px;
+  margin-left: auto;
+  margin-right: auto;
+}
```
Comment (TheJaredWilcurt) [inline style.css]: End files with an empty return to appease git.
```suggestion
}

```

**Pull request merged**